src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbBasicTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsde.rawls.graph
```

Comment:
sorry, I was talking about the Test. Do any of the Tests in here matter? Do we need this file at all?